resolvers/webpack/index.js

```diff
@@ -265,7 +266,7 @@ function findExternal(source, externals, context) {
 
   // else, vanilla object
   for (var key in externals) {
-    if (!externals.hasOwnProperty(key)) continue
+    if (!has(externals, key)) continue
```

Comment:
Agh, nevermind, it's bundled with some other stuff. I'll just keep it for now. 😁